.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+exclude: |
+  (?x)^(
+    .*third_party/.*|
+    .*laf/.*|
+  )
```

Comment:
I've tested on Linux and it looks like all .cpp files are being excluded, changed this to
```suggestion
exclude: 'third_party/.*|laf/.*'
```
and it started to work.